src/compiler/core.ts

```diff
@@ -1705,14 +1683,9 @@ export function createSet<TElement, THash = number>(getHashCode: (element: TElem
             const hash = getHashCode(element);
             if (!multiMap.has(hash)) return false;
             const candidates = multiMap.get(hash)!;

```

Comment:
Does this utility function expressly support `undefined` being a meaningful value in the set? Otherwise this would be more efficient to call `get` and do a check against `undefined` instead of calling into the map twice.